crates/core/tedge_mapper/src/core/converter.rs

```diff
@@ -67,6 +71,13 @@ pub trait Converter: Send + Sync {
     fn sync_messages(&mut self) -> Vec<Message> {
         vec![]
     }
+
+    fn process_operation_update_messages(
+        &mut self,
+        _message: DiscoverOp,
+    ) -> Result<Message, ConversionError> {
+        Ok(Message::new(&make_valid_topic_or_panic(""), ""))
+    }
```

Comment:
Please, no code like this one :-(

If some converter implementation can return no message, then use the `Option` type. If some implementation might have to return many messages, use `Vec` as done for `try_convert`.

Please, also note that the names of this trait follow a pattern. Take the time to read the code you update.

Reply:
Fixed, using Option<Message>